sendgrid.go

```diff
@@ -125,3 +125,12 @@ func MakeRequestAsync(request rest.Request) (chan *rest.Response, chan error) {
 
 	return r, e
 }
+
+// CheckSecrets checks email content for presence of an API key before sending.
+func CheckSecrets(mailContent string) (int, error) {
+	var secret = regexp.MustCompile(`SG.[a-zA-Z0-9_-]+.[a-zA-Z0-9_-]+`)
```

Comment:
Consider creating a global variable for this regex so we don't have to initialize in on every `CheckSecrets` call. It could be called `apiKeyRegex`.